.gitignore

```diff
@@ -51,10 +51,12 @@ yarn-error.log*
 .tool-versions
 
 # Allow each developer to have personal settings, but ignore these files
-.vscode/launch.json 
+.vscode/launch.json
 .vscode/*.settings.json
 
 .sonar
 .sonar/**/*
 
-dependency-graph.svg
\ No newline at end of file
+dependency-graph.svg
+logs/
+__pycache__/
```

Comment:
Fine with this being in here but there's not supposed to be any Python in the codebase (although I did accidentally add one file before which is still in here)